### DISCUSSION TITLE
MinIO tiering

### DISCUSSION THREAD

**Discussion by ghost** [minio/minio#18742] [Q&A] [answered]
Hi. Can I know why the dropdown is empty? Is there any guide that I can refer to set up the tiering step by step in minIO console? What are the possible values to be included in that tier values?
![minio](https://github.com/minio/minio/assets/155696966/bd885a5e-bfe7-41ad-871a-c0f0dc4d7615)

I already run command: mc admin tier add minio local STANDARD_IA --endpoint http://xxxx --access-key xxx --secret-key xxx --bucket bucket-test --prefix warm and got this appear in my console. Is my setup for tiering wrong?
![image](https://github.com/minio/minio/assets/155696966/21828633-49ce-4e64-aee9-8692a73a8b63)

**Comment by wuyanfeiwork** [comment, marked as answer]
![image](https://github.com/minio/minio/assets/104751498/8ede7121-ded1-4641-ab23-916813e4fdd0)

You need to create tier.

**Reply by ghost** [reply]
Hi, I already set like this but got some error. Can I know what is wrong with the setup as even though it has error, it created the folder name and appear the 'probeobject' but the tiering still shows nothing.

![image](https://github.com/minio/minio/assets/155696966/bdb39fd2-a1b3-4499-a8e9-2855ba05bc1e)

![image](https://github.com/minio/minio/assets/155696966/e82b8220-9a3c-4537-9d3b-d0d7901108a2)

**Reply by harshavardhana** [reply]
Your tier is unreachable

**Reply by ghost** [reply]
May I know what are the possible cause? Is there any other place that I need to setup first?

**Reply by klauspost** [reply]
> May I know what are the possible cause?

That the remote isn't reachable from your cluster. Check your network setup.

**Reply by ghost** [reply]
In my case, the minio is deploy inside the docker and we are accessing the minio url using VM IP. Could this be the cause of the tier  is unreachable? Is there any solution for this issue to setup the tiering?
